ext/intl/timezone/timezone_methods.cpp

```diff
@@ -150,9 +148,8 @@ U_CFUNC PHP_FUNCTION(intltz_create_enumeration)
 int_offset:
 		if (UNEXPECTED(Z_LVAL_P(arg) < (zend_long)INT32_MIN ||
 				Z_LVAL_P(arg) > (zend_long)INT32_MAX)) {
-			intl_error_set(NULL, U_ILLEGAL_ARGUMENT_ERROR,
-				"value is out of range", 0);
-			RETURN_FALSE;
+			zend_argument_value_error(1, "value is out of range");
```

Comment:
Maybe say the integer must fit in a 32-bit signed integer? Also adding a test would be great :)